tools/esdlcmd/esdlcmd_core.cpp

```diff
@@ -521,6 +521,21 @@ class Esdl2WSDLCmd : public Esdl2XSDCmd
         cmdHelper.loadDefinition(optSource, optService.get(), optInterfaceVersion, "", optTraceFlags());
         createOptionals();
 
+        if (optInterfaceVersion == 0.0)
```

Comment:
This change looks good, but it may be a good idea to add the same logic to the `Esdl2XSDCmd` for consistency of behavior.